lib/modules/manager/leiningen/extract.ts

```diff
@@ -36,7 +37,8 @@ export function extractFromVectors(
   let vecPos = 0;
   let artifactId = '';
   let version = '';
-  let commentLevel = 0;
+  // Are we currently parsing a comment? If so, at what depth?
+  let commentLevel : number | null = null;
```

Comment:
I made this change to be able to distinguish between "we are not parsing a comment" and "we are in a comment at level 0". Otherwise when `dimension = 1`, the condition on line 93 would be hit, causing the dependency info to be thrown away.